aperturedb/transformers/facenet.py

```diff
@@ -0,0 +1,25 @@
+from facenet_pytorch import MTCNN, InceptionResnetV1
+from PIL import Image
+import torch
```

Comment:
if we aren't going to require these modules in requirements we might want to wrap the import with a try

Reply:
fair point. will do so.